src/PartitionGraph.f90

```diff
@@ -2209,4 +2229,187 @@ SUBROUTINE swapDomain(thisGraph,iv,linL1,D,lswap)
       IF(lswap) linL1(iv)=(.NOT. linL1(iv))
     ENDSUBROUTINE swapDomain
 !
+!-------------------------------------------------------------------------------
+!> @brief A fixup for discontiguous domains during recursive partitioning
+!> @param thisGraph The graph object
+!> @param Order The order vector
+!> @param nv1 The number of vertices in sub-domain 1
+!> @param lContiguous Logical for if the two subdomains are contiguous
```

Comment:
Not a parameter in this routines interface

Reply:
Ah, good catch! I changed the interface and failed to update the documentation.